source/Halibut.Tests/Support/TestPollingClient.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace Halibut.Tests.Support
+{
+    public class TestPollingClient : ITestPollingClient
```

Comment:
nit: why do we need `ITestPollingClient`?

Reply:
Agreed. Deleted